src/main/kotlin/racingcar/ui/ResultView.kt

```diff
@@ -0,0 +1,42 @@
+package racingcar.ui
+
+import racingcar.domain.model.CarRacer
+
+class ResultView {
+
+    fun show(result: List<List<CarRacer>>) {
+        println("")
+        println("실행 결과")
+        result.forEachIndexed { index, gameResult ->
+            showGameResult(gameResult)
+            showWinners(result.lastIndex, index, gameResult)
+        }
+    }
+
```

Comment:
😱 모든 루프에서 마지막 우승자 출력을 시도하는데요, 우선 `showGameResult`만 모두 수행하고, 마지막에 한번만 우승자 출력을 시도하면 어떨까요?
```suggestion
    fun show(result: List<List<CarRacer>>) {
        println("")
        println("실행 결과")
        result.forEach(::showGameResult)
        showWinners(result.last())
    }
```

또한 이렇게 되면 실행결과와 우승자 출력 메서드를 분리할 수도 있겠죠?

```suggestion
fun show(result: List<List<CarRacer>>) {
        showGameResult(result)
        showWinners(result.last())
    }
```

Reply:
이 부분은 좀 더 개선이 필요한 코드였습니다! 
의견 반영하여 수정합니다!